manifests/http_port.pp

```diff
@@ -1,24 +1,43 @@
 define squid::http_port (
-  Variant[Pattern[/\d+/], Integer]
-          $port    = $title,
-  Boolean $ssl     = false,
-  String  $options = '',
-  String  $order   = '05',
+  Optional[Integer] $port    = undef,
+  Optional[String]  $host    = undef,
```

Comment:
Can you please use `Optional[String[1]] `? This enforces a minimal string length of 1.